windows-driver-docs-pr/image/creating-a--hello-world--wia-minidriver-ui-extension.md

```diff
@@ -100,7 +100,7 @@ HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C6}\InProcServer32,,,%11%\hellowld
 HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C6}\InProcServer32,ThreadingModel,,Both
 
 HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C7},,,"Hello World WIA Minidriver UI Extension"
-HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C7}\InProcServer32,,,%11%\hellowldui.dll
+HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C7}\InProcServer32,,,%%SystemRoot%%\hellowldui.dll
```

Comment:
Due to you changing DefaultDestDir above to 13, this file won't exist at %systemroot%\hellowldui.dll.  This should be %13%\hellowldui.dll

Reply:
also, this line a few lines above also needs updating:
HKCR,CLSID\{7C1E2309-A535-45b1-94B3-9A020EE600C6}\InProcServer32,,,%11%\hellowld.dll